integration/spark/src/main/scala/org/apache/spark/sql/execution/command/management/CommonLoadUtils.scala

```diff
@@ -1064,6 +1064,7 @@ object CommonLoadUtils {
         if (loadParams.updateModel.isDefined) {
           CarbonScalaUtil.updateErrorInUpdateModel(loadParams.updateModel.get, executorMessage)
         }
```

Comment:
if possible, Please add a small testcase that intercepts exception with proper error message

Reply:
Added testcase